### PR TITLE
Changes references to RCv4 back to `main`

### PR DESCRIPTION
**Pull request by taquitos**

### CODE REVIEW

File: IntegrationTests/CarthageIntegration/purchases-root [hunk 1]
```diff
@@ -1 +0,0 @@
-../../
```
Comment (vegaro) [inline IntegrationTests/CarthageIntegration/purchases-root:1]: do we need this file or it can be removed?

File: .circleci/config.yml [hunk 2]
```diff
@@ -314,7 +314,7 @@ jobs:
           # install without building, then remove the tests and build, so that carthage
           # doesn't try to build the other integration tests
           command: |
-              ./carthage.sh update --no-build
+              ./carthage.sh bootstrap --no-build
```
Comment (vegaro) [inline .circleci/config.yml:317]: Is this change intended? It used to be `update` https://github.com/RevenueCat/purchases-ios/commit/caf58d14b715bf0b510f93e5fd7cae6dab912784#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R317

File: IntegrationTests/CarthageIntegration/purchases-root (hunk 1 above)
Comment (taquitos) [reply IntegrationTests/CarthageIntegration/purchases-root:1]: I needed to update that. Changing back to what it was.

File: .circleci/config.yml (hunk 2 above)
Comment (taquitos) [reply .circleci/config.yml]: Yup, initially I thought this was better, and it Worked On My Machine™️
